src/Illuminate/Support/helpers.php

```diff
@@ -507,6 +508,39 @@ function dd()
     }
 }
 
+if (! function_exists('dddb')) {
+    /**
+     * Dump last query and exit.
+     * Needs query logging to be enabled.
```

Comment:
Not sure I'm understanding you here - the diff has shown the additional newline at the bottom of the added code, not the top. Is that what you're looking for?